tests/units/Glpi/Form/Form.php

```diff
@@ -50,6 +50,32 @@ class Form extends DbTestCase
 {
     use FormTesterTrait;
 
+    /**
+     * Some questions require extra data to be rendered
+     * This provider provides extra data for each question type
+     *
+     * @return array
+     */
+    public function renderProvider(): array
```

Comment:
You are hijacking the provider feature, and it is probably not a good idea. Indeed a test provider is supposed to be used to trigger multiple calls to the test methods, but with different parameters for each call.

Here you just need hardcoded values for some question types. You should move these hardcoded values directly inside the test method.